py/torch_tensorrt/dynamo/converter_registry.py

```diff
@@ -0,0 +1,305 @@
+from dataclasses import dataclass, field
+from typing import Any, Callable, Dict, Optional, Sequence, Union
+
+from torch.fx.node import Target, Node, _get_qualified_name
+from torch_tensorrt.fx.converter_registry import CONVERTERS
+
+
+@dataclass(frozen=True)
+class ConverterSupport:
+    """Class representing a converter implementation and support function
+
+    Args:
+        converter_implementation: Function which converts said node to a TRT equivalent
+        capability_validator: Function which takes in a Node and returns a bool indicating
+            whether that node can be supported by its companion converter. Note that
+            this function must not modify the node or its graph
+    """
+
+    converter_implementation: Callable
+    capability_validator: Callable[[Node], bool] = field(default=lambda node: True)
+
+
+# Dictionary representing Dynamo aten-only converters
+# Each converter maps to a sequence of at least one ConverterSupport object(s)
+DYNAMO_ATEN_CONVERTERS: Dict[Target, Sequence[ConverterSupport]] = {}
+
+
+def dynamo_tensorrt_converter(
+    key: Target,
+    enabled: bool = True,
+    capability_validator: Optional[Callable[[Node], bool]] = None,
+    high_priority: bool = False,
```

Comment:
Added support for high-priority converters to be prioritized over others during compilation

Reply:
Make `priority` flag an Enum `{HIGH, STANDARD}`